src/lisp/regression-tests/system-construction.lisp

```diff
@@ -49,7 +49,7 @@
 ;;; crosscompiling sbcl
 (test-true compile-file-serial-no-faso
  (let ((cmp::*compile-file-parallel* nil)
-       (core:*clasp-build-mode* :faso)
+       (cmp:*default-output-type* :faso)
```

Comment:
it might be better to use the `:output-type` parameter to `compile-file` rather than the special variable.

Reply:
Done